src/components/Menus/WalletMenu/WalletMenu.tsx

```diff
@@ -41,45 +45,67 @@ export const WalletMenu = ({ anchorEl }: WalletMenuProps) => {
   }, [accounts, setWalletMenuState, openWalletMenu]);
 
   return (
-    <Menu
+    <Drawer
+      variant="persistent"
+      anchor="right"
       open={openWalletMenu}
-      setOpen={setWalletMenuState}
-      isOpenSubMenu={openSubMenu !== MenuKeysEnum.None}
-      width={'auto'}
-      styles={{
-        background: theme.palette.surface1.main,
-        padding: '12px',
+      onClose={() => {
+        setWalletMenuState(false);
+        setOpen(false);
+      }}
+      PaperProps={{
+        sx: (theme) => ({
+          width: '100%',
```

Comment:
why not create a component?

Reply:
I think its gotta be like that to pass styles to the paper-element of the Drawer component.